src/index.ts

```diff
@@ -9,7 +9,9 @@ export function transformSync(
 	source: string,
 	options: Options,
 ): TransformOutput {
-	return swc.transformSync(source, {
+	// Ensure that the source code is a string
+	const input = !source || typeof source !== "string" ? "" : source;
```

Comment:
I think a more proper type conversion could be:

```suggestion
	const input = `${source ?? ''}`;
```

Reply:
Doesn't work weel 
```js 
const foo = {};
`${foo}`
// '[object Object]'
```